src/clr-angular/data/datagrid/datagrid-row.spec.ts

```diff
@@ -26,23 +27,24 @@ import {StateDebouncer} from "./providers/state-debouncer.provider";
 import {DomAdapter} from "./render/dom-adapter";
 import {DatagridRenderOrganizer} from "./render/render-organizer";
 
+
 const PROVIDERS = [
     Selection, Items, FiltersProvider, Sort, Page, RowActionService, ExpandableRowsCount, DatagridRenderOrganizer,
-    DomAdapter, HideableColumnService, DatagridWillyWonka, StateDebouncer
+    DomAdapter, HideableColumnService, DatagridWillyWonka, StateDebouncer, DisplayModeService
 ];
 
 export default function(): void {
```

Comment:
Missing a number of tests

* handles expand changes
* host wrapping tests
* display mode changes

Reply:
Added tests for these three areas.